script/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
What does this file do?

Reply:
This is the docker command entry point and it's also meant to fix an issue with starting rails applications in a docker container described here:
https://docs.docker.com/compose/rails/#define-the-project